### DISCUSSION TITLE
A new beginning

### DISCUSSION THREAD

**Discussion by jdev082** [CatalystDevOrg/Catalyst#2] [General]
Discuss the future...

**Comment by hello-smile6** [comment]
Could this be a github org?

**Reply by jdev082** [reply]
NO! I manage the repository and flow of PRs.
I know what mess you made last time, that's why I'm incorporating long reading and aproving of PRs before merge.

**Reply by hello-smile6** [reply]
> NO! I manage the repository and flow of PRs. I know what mess you made last time, that's why I'm incorporating long reading and aproving of PRs before merge.

I'm just suggesting a 1-person org with just you. Then there could be multiple repositories, one for each component, kinda like @turbowarp

**Reply by jdev082** [reply]
@hello-smile6 repositories for each component is stupid

**Reply by hello-smile6** [reply]
> @hello-smile6 repositories for each component is stupid

So, you'd rather have your css framework in the same repo?

**Reply by jdev082** [reply]
@hello-smile6 the framework has its own repo, and it just has an implementation in Catalyst. I've moved Catalyst to Material UX though.

**Reply by imdevarsh** [reply]
@JaydenDev which framework?

**Reply by jdev082** [reply]
@webdev03 ZenCSS which is no longer used.
plus the switch to tailwind has been done. this discussion is outdated
